### PR TITLE
[`Blip`] Fix blip output name

### PR DESCRIPTION
**Pull request by younesbelkada**
As suggested by @ydshieh offline and similarly as https://github.com/huggingface/transformers/pull/22893
In fact, there is no reason to call the output logits `decoder_logits`, as they always come from the decoder

cc @sgugger @ydshieh

### CODE REVIEW

**Comment by younesbelkada** [comment]
Thanks @ydshieh for double checking, I missed that, it should be now added! :D

**Review by ydshieh — APPROVED** [review]
Thanks for taking care of our codebase!

**Review by sgugger — APPROVED** [review]
Makes sense, thanks!
